src/System.Management.Automation/engine/Utils.cs

```diff
@@ -1873,6 +1873,113 @@ private static bool TryGetCommandInfoList(PowerShell ps, HashSet<string> command
             return true;
         }
 
+        internal static OutputRendering OutputRenderingSetting = OutputRendering.Automatic;
```

Comment:
If we go with the singleton implementation of `PSStyle`, then the setting can just live with that singleton and we can remove this static filed.

Reply:
@SteveL-MSFT It seems you missed this comment.

Reply:
missed this, will update